pesquini/app/views/layouts/_header.html.erb

```diff
@@ -4,7 +4,7 @@
   <div class="container">
     <!-- Brand and toggle get grouped for better mobile display -->
     <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+      <button type="but" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
```

Comment:
Porque na welcome index já tem um butão com o nome "button " e o teste não passa dessa maneira. Por isso mudei o nome para "but" na header.

Date: Sun, 17 May 2015 11:08:49 -0700
From: notifications@github.com
To: pesquini@noreply.github.com
CC: ruandonato@outlook.com
Subject: Re: [pesquini] Pull Request teste_unitario (#6)

In pesquini/app/views/layouts/_header.html.erb:

> @@ -4,7 +4,7 @@
>    <div class="container">
>      <!-- Brand and toggle get grouped for better mobile display -->
>      <div class="navbar-header">
> -      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
> +      <button type="but" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">

Type but?


—
Reply to this email directly or view it on GitHub.